editor.planx.uk/src/routes/views/flowEditor.tsx

```diff
@@ -72,9 +67,9 @@ const getFlowMetadata = async (
  */
 export const flowEditorView = async (req: NaviRequest) => {
   const [flow] = req.params.flow.split(",");
-  const { flowSettings, flowAnalyticsLink, isFlowPublished } =
+  const { flowAnalyticsLink, isFlowPublished } =
     await getFlowMetadata(flow, req.params.team);
-  useStore.setState({ flowSettings, flowAnalyticsLink, isFlowPublished });
+  useStore.setState({ flowAnalyticsLink, isFlowPublished });
```

Comment:
`flowSettings` is not needed here - it's only actually used and displayed when we navigate to the settings page.